### PR TITLE
gopls 0.5.3 (new formula)

### PR DESCRIPTION
**Pull request by xsteadfastx**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/gopls.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  depends_on "jsonlint" => :test
+
+  def install
+    Dir.chdir("gopls") do
+      system "go", "build", *std_go_args
+    end
+  end
+
+  test do
+    system "#{bin}/gopls api-json | jsonlint"
```
Comment (dawidd6) [inline Formula/gopls.rb:18]: Can we check the output here?

File: Formula/gopls.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  homepage "https://github.com/golang/tools/tree/master/gopls"
+  url "https://github.com/golang/tools/archive/gopls/v0.5.3.tar.gz"
+  sha256 "f3a1c142f42c7e9cff62ec9a1a6e41e59305c7519735839ecedccf17597ae00e"
+  license "BSD-3-Clause"
+
+  depends_on "go" => :build
+  depends_on "jsonlint" => :test
+
+  def install
+    Dir.chdir("gopls") do
```
Comment (dawidd6) [inline Formula/gopls.rb:12]:
```suggestion
    cd "gopls" do
```

File: Formula/gopls.rb (hunk 1 above)
Comment (xsteadfastx) [reply Formula/gopls.rb:18]: Its a really large Output... Alot of lines... Would that still be OK?

Comment (dawidd6) [reply Formula/gopls.rb:18]: Just check for a part of it. With `assert_match` for example.

File: Formula/gopls.rb (hunk 2 above)
Comment (xsteadfastx) [reply Formula/gopls.rb]: thanks :)

File: Formula/gopls.rb (hunk 1 above)
Comment (xsteadfastx) [reply Formula/gopls.rb]: i took now a little bit of the json and did a `assert_match`. i hope its ok like this.

Comment (dawidd6) [reply Formula/gopls.rb]: If this command outputs a valid JSON document, you can parse it with `JSON.parse` and then check some hash fields and values.

Comment (xsteadfastx) [reply Formula/gopls.rb]: done that... what do you think?

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@xsteadfastx, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**